bin/qds.py

```diff
@@ -510,6 +518,10 @@ def main():
                          default=False,
                          help="skip verification of server SSL certificate. Insecure: use with caution.")
 
+    optparser.add_option("--cloud", dest="cloud",
+                         default=os.getenv('QDS_CLOUD'),
+                         help="cloud", choices=["aws", "azure", "oracle_bmc"])
```

Comment:
@sourabh912 Should this be in capital since these are being set from an environment variable?